Vagrantfile

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# This script provisions a vagrant development environment with local
+# postgres and redis servers for development and testing.
+PROVISION = <<SCRIPT
+set -e
+
+# cd to /vagrant on login
+cd /vagrant
+grep -q 'cd /vagrant' ~/.bashrc || echo 'cd /vagrant' >> ~/.bashrc
+
+# Install system packages
+sudo apt-get update --quiet
+sudo apt-get install -y $(cat debian_packages.lst) postgresql
+
+# Upgrade pip
+sudo pip3 install --upgrade pip
+
+# By default, pip will install editable packages in ./src/ which is inside
+# the virtualbox share. This can slow things like prospector down a lot, so
+# configure pip to install them at the usual dist-packages location instead.
+cat << EOF | sudo tee /etc/pip.conf
+[install]
+src = /usr/local/lib/python3.4/dist-packages
+EOF
+
+# Install python dependencies
+sudo pip install -r requirements.txt
+
+# Create postgres user
+sudo -u postgres createuser -d vagrant
+
+# Allow access to postgres from localhost without password
+cat << EOF | sudo tee /etc/postgresql/9.3/main/pg_hba.conf
+local   all             postgres                                peer
+local   all             all                                     trust
+host    all             all             127.0.0.1/32            trust
+host    all             all             ::1/128                 trust
+EOF
+sudo service postgresql restart
```

Comment:
@omarkhan It's not the case by default? Honest question, I don't remember having to relax the default auth on my Ubuntu - but the defaults might have changed since I installed it (or I might incorrectly remember it).